nautobot/docs/development/getting-started.md

```diff
@@ -89,45 +187,44 @@ SUPERUSER_API_TOKEN=0123456789abcdef0123456789abcdef01234567
 ```
 
 !!! warning
-    Please name the **.env** file ``override.env`` to prevent credentials from being checked into Git. ``override.env`` is set in the ``.gitignore`` file.
+    Please name the **.env** file ``override.env`` to prevent credentials from accidentally being checked into Git as ``override.env`` is set in the ``.gitignore`` file.
 
-These will create the user with the specified username, email, password, and API token.
+These variables will tell `docker-entrypoint.sh` to create the superuser with the specified username, email, password, and API token.
```

Comment:
The `override.env` variables will signal to that `docker-entrypoint.sh`shell script to create ....